docs/BreakingChanges/047 WinRT stream adapters no long call FlushAsync auto.md

```diff
@@ -0,0 +1,30 @@
+## 47: WinRT stream adapters no long call FlushAsync automatically on close
```

Comment:
{
  "Id": "47",
  "Title": "WinRT stream adapters no long call FlushAsync automatically on close",
  "Details": "In Windows Store apps, Windows Runtime stream adapters no longer call the FlushAsync method from the Dispose method.",
  "Suggestion": "This change should be transparent. Developers can restore the previous behavior by writing code like this:\n\n```csharp\nusing (var stream = GetWindowsRuntimeStream() as Stream)\n{\n// do something\nawait stream.FlushAsync();\n}\n```",
  "Link": "https://msdn.microsoft.com/en-us/library/dn458360(v=vs.110).aspx",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5.1",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 4
}

**Parsed APIs:**
- [ ] Not detectable via API analysis